packages/cli-types/src/commands/doctor/index.ts

```diff
@@ -0,0 +1,79 @@
+// @flow
```

Comment:
remove flow

Reply:
Btw, how about putting the types into `doctor/types.ts`, similar as we have with Flow ones? In general, `index.js`/`index.ts` should only be used to actually "index" (list) some files/variables to export